assets/data_table/main.js

```diff
@@ -303,14 +346,18 @@ function Pagination({ page, maxPage, onPrev, onNext }) {
   );
 }
 
-function reorder(orderBy, order, key) {
-  if (orderBy === key) {
-    if (order === "asc") {
-      return [key, "desc"];
-    } else {
-      return [null, "asc"];
-    }
-  } else {
-    return [key, "asc"];
-  }
+function HeaderMenu({ layerProps, orderBy }) {
+  return (
+    <div className="header-menu" {...layerProps}>
+      <div className="header-menu-item" onClick={() => orderBy("asc")}>
+        ascending
+      </div>
+      <div className="header-menu-item" onClick={() => orderBy("desc")}>
+        descending
+      </div>
+      <div className="header-menu-item" onClick={() => orderBy(null)}>
+        none
```

Comment:
We will likely add other commands later. Should this say:

```suggestion
        Sort: ascending
      </div>
      <div className="header-menu-item" onClick={() => orderBy("desc")}>
        Sort: descending
      </div>
      <div className="header-menu-item" onClick={() => orderBy(null)}>
        Sort: none
```

Additionally we could use icons, but I think it is probably overkill?

Reply:
Well, we already have icons on the header and soon we'll have the summaries. I'm not sure we should add more icons... I think it would be too much information